lib/iris/coords.py

```diff
@@ -1064,28 +1087,25 @@ def is_contiguous(self, rtol=1e-05, atol=1e-08):
 
         """
         if self.has_bounds():
-            self._sanity_check_contiguous()
+            self._sanity_check_bounds()
             if self.ndim == 1:
                 contiguous = np.allclose(self.bounds[1:, 0],
                                          self.bounds[:-1, 1],
                                          rtol=rtol, atol=atol)
             elif self.ndim == 2:
-                contiguous, _, _ = _discontiguity_in_2d_bounds(self.bounds,
-                                                               abs_tol=atol)
+
+                contiguous, _, _ = _discontiguity_in_2d_bounds(
+                    self.bounds, atol=atol)
         else:
             contiguous = False
         return contiguous
 
     def contiguous_bounds(self):
         """
```

Comment:
Docstring is better (I like things which are concise) but I'm still not taken with the name.  'contiguous_bounds' is just a bit vague.

Reply:
Unfortunately we don't have much say with changing the name of `contiguous_bounds`. It was already part of the API so changing it would be a breaking change.